Document Goshimmer API (#1342)
iotaledger/goshimmer: docs/apis/info.md

> # Info API

Reviewer: ```suggestion
# Info API Methods
```

Info API returns basic info about the node

The API provides the following functions and endpoints:

* [/info](#getinfo)

Client lib APIs:
* [Info()](#getinfo)

> ##  `/autopeering/neighbors`

Reviewer: ```suggestion
##  `/info`
```

Returns basic info about the node.

### Parameters
None.

### Examples

#### cURL

```shell
curl --location 'http://localhost:8080/info'
```

#### Client lib

> Messages can be retrieved via `Info() (*jsonmodels.InfoResponse, error)`

Reviewer: ```suggestion
Information of a node can be retrieved via `Info() (*jsonmodels.InfoResponse, error)`
```

> ```

Reviewer: ```suggestion
```go
```

    "Logger",
    "Mana",
    "ManaRefresher",
    "Manualpeering",
    "MessageLayer",
    "Metrics",
    "NetworkDelay",
    "PoW",
    "PortCheck",
    "Profiling",
    "Prometheus",
    "RemoteLog",
    "RemoteLogMetrics",
    "WebAPI",
    "WebAPI DRNG Endpoint",
    "WebAPI Mana Endpoint",
    "WebAPI WeightProvider Endpoint",
    "WebAPI autopeering Endpoint",
    "WebAPI data Endpoint",
    "WebAPI faucet Endpoint",
    "WebAPI healthz Endpoint",
    "WebAPI info Endpoint",
    "WebAPI ledgerstate Endpoint",
    "WebAPI message Endpoint",
    "WebAPI tools Endpoint",
    "snapshot"
  ],
  "disabledPlugins": [
    "Analysis-Dashboard",
    "Analysis-Server",
    "Faucet",
    "ManaEventLogger",
    "Spammer",
    "TXStream"
  ],
  "mana": {
    "access": 1,
    "accessTimestamp": "2021-05-24T20:11:05.451224937+02:00",
    "consensus": 10439991680906,
    "consensusTimestamp": "2021-05-24T20:11:05.451228137+02:00"
  },
  "manaDelegationAddress": "1HMQic52dz3xLY2aeDXcDhX53LgbsHghdfD8eGXR1qVHy",
  "mana_decay": 0.00003209,
  "scheduler": {
    "running": true,
    "rate": "5ms",
    "nodeQueueSizes": {}
  },
  "rateSetter": {
    "rate": 20000,
    "size": 0
  }
}
```

> #### Results

Reviewer: Add tables for `TangleTime`, `Scheduler` and `RateSetter`.
